### DISCUSSION TITLE
Future of code for I

### DISCUSSION THREAD

**Discussion by pabloto** [codefori/vscode-ibmi#1030] [Q&A] [answered]
Hi Liam, 

First of all Great job!!!

I'm using code for I in local, with git integrations and I love it. I still have to understand the integration with bob, (do you have some tutorials about it?), but to me is perfect. 

Now here's my problem, I want to extend the use of it to all my colleagues (they still use seu, and a lot of programs are still in fixed form with move, z-add, goto😱😱😱). Their excuse is that code for I is not Ibm officials. If you left the project, and I hope it never will happen, probably no one could take your heritage... 

What's the future of code for I?

Someone in the world have started to use Merlin? I saw only some presentation, but it seems that requires a lot of knowledge...to us that comes from seu, Code for I is just something like "back to the future".

How can I help my colleagues to move to code for I?

Many thanks 

Regards!

**Comment by edmundreinhardt** [comment, marked as answer]
Hello,
My name is Edmund Reinhardt, and I am an IBMer who has been working on application development tools since 1989 when I started with SDA.  As a technical architect of IBM I developer tools, including RDi, Merlin and BOB, I can tell you that Code for I is strategic for IBMi.  I have made my own contributions to it, and as we develop Merlin technology we are looking for investments that will benefit both Merlin and the Code for I communities.  Why is IBM doing this?  Because we are not just interested in selling tools we are interested in our customers succeeding on this platform.  We strongly feel that the use of SEU and fixed form is holding back the productivity of our customers and preventing them from taking advantage of great young talent that could easily contribute given a free form RPG and a modern IDE.  We recognize that not everyone can afford the fuller function RDi or Merlin, but with Code for i, cost is no longer a barrier.  Please encourage your colleagues, that IBMer's like Tim Row, Edmund Reinhardt, Jesse Gorzinski as well as our beloved Liam are investing in Code for i as a strategic solution for our platform, so there is no excuse to stay with SEU.
We want no programmer left behind.

Edmund Reinhardt
